### ISSUE TITLE
`Set-PsFzfOption` `PSReadlineChordReverseHistory` - Multi Line Command

### ISSUE THREAD

**Issue by mattcargile** [kelleyma49/PSFzf#137]
I want to look more at the `.dll` but I don't think it handles multi line commands. I think there needs new `if` logic to handle the backtick ( <code>\`</code> ) on particular lines within the `ReverseLineReader.cs`. Below is the way the Microsoft folks do it in the `PSReadline` repo sample.

https://github.com/PowerShell/PSReadLine/blob/dc38b451bee4bdf07f7200026be02516807faa09/PSReadLine/SamplePSReadLineProfile.ps1#L43

Here is an example screenshot.

![image](https://user-images.githubusercontent.com/74641867/158938278-3a8ef322-9903-447b-9bee-afc7892a05b9.png)

**Comment by kelleyma49** [comment]
Good catch. It doesn't currently look at mult-line commands. Let me see how easy this is to add.

**Comment by snowman** [comment]
![read-history-of-multiple-line-command](https://user-images.githubusercontent.com/16096559/173834839-e7a1d657-a973-41dc-a87b-d582458bdd91.jpg)

It would be awesome if `-PSReadlineChordReverseHistory` option shows the multiple-line command as a single line with `fzf`, but show the details(full multiple-lines) within the sidebar preview

**Comment by mattcargile** [comment]
I'm going to attempt to touch this now.

**Comment by jshall** [comment]
Has there been any progress on this? @mattcargile is your attempt available in a branch that I could attempt to iterate upon?

**Comment by mattcargile** [comment]
I haven't had a chance, @jshall . I don't have a branch either as I was still working on understanding how reading a file in reverse works.

**Comment by kelleyma49** [comment]
Give this release a try: 

https://www.powershellgallery.com/packages/PSFzf/2.6.8-alpha

**Comment by kelleyma49** [comment]
@mattcargile or @jshall , let me know if it works for you!

**Comment by jshall** [comment]
> [@mattcargile](https://github.com/mattcargile) or [@jshall](https://github.com/jshall) , let me know if it works for you!

Thank you!

The search, display and selection work well, but when the selection is returned to the command line the line endings are replaced with spaces. This often results in a non-viable command as I omit `;` when using multiple lines.

**Comment by jshall** [comment]
It looks like this can be fixed by changing the `$result` at the end of Get-PickedHistory with ``$result -join "`n"``.
